pytorch_toolkit/ote/ote/api.py

```diff
@@ -67,6 +95,7 @@ def test_args_parser(template_path):
                             help='Location where output images (with displayed result of model work) will be stored.')
         parser.add_argument('--config', default=config['config'], help=argparse.SUPPRESS)
 
+        _add_nncf_arguments_to_parser(parser, config)
```

Comment:
I don't like adding this args to test and export script. From my perspective - keeping compression info in a snapshot is better.